From 5959167df5167bcc25b6dddec1513b0d63eeda69 Mon Sep 17 00:00:00 2001
From: Maximilian Sackel <maximilian.sackel@udo.edu>
Date: Thu, 6 May 2021 14:48:37 +0200
Subject: [PATCH] :bug: add missing string header caused windows build to crash

---
 src/PROPOSAL/PROPOSAL/methods.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/PROPOSAL/PROPOSAL/methods.h b/src/PROPOSAL/PROPOSAL/methods.h
index 34f1f1d0..3e48b909 100644
--- a/src/PROPOSAL/PROPOSAL/methods.h
+++ b/src/PROPOSAL/PROPOSAL/methods.h
@@ -28,7 +28,7 @@
 
 #pragma once
 
-#include <deque>
+#include <string>
 #include <functional>
 #include <map>
 #include <memory>
-- 
2.31.0

